src/components/nav.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&family=Roboto+Slab:wght@200;400;600&display=swap");
+
+.nav1 {
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.navbar {
+  display: flex;
+  width: 84.5%;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.navList {
+  display: flex;
+  gap: 80px;
+}
+
+.nav2 {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.nav3 {
+  color: #d2d2d2;
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+}
```

Comment:
- [ ] This link is a little difficult to find as its color is too similar to the background. Let's try to increase the contrast of it so at least matches the contrast on the template (Which is not much either, but it's better).

Yours:
![image](https://user-images.githubusercontent.com/79114233/146472723-88ae3c12-f356-44aa-8d56-f14eb634288e.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/146472737-be4445c6-b577-4824-a445-e7e56975a307.png)

[OPTIONAL] Usually when using `CSS` or `SASS` we want to be very specific with the naming of the components. That's for a couple of reasons, the first one, if we are too generic we might replace the styling later on as two components might get the same name; the second one is making our code more readable for us and others. (Coming back to projects that we've developed in the past it's painful for this reason)

I would name this class something like this: `nav__link`. This follows a naming convention named BEM, which stands for Block, Element, and Modifiers (`block__element--modifier`). [Read more about BEM here.](http://getbem.com/introduction/) :sweat_smile:

Reply:
@D4nielJ Thanks for your valuable feedback, it was really helpful.